src/data/mv_polynomial/basic.lean

```diff
@@ -913,6 +934,22 @@ begin
   refl
 end
 
+/-- If `f : S₁ →ₐ[R] S₂` is an morphism of `R`-algebras, then so is `mv_polynomial.map f`. -/
```

Comment:
I'm not sure a one-character change like this is worth the CI time but here is the suggestion anyway:
```suggestion
/-- If `f : S₁ →ₐ[R] S₂` is a morphism of `R`-algebras, then so is `mv_polynomial.map f`. -/
```